index.html

```diff
@@ -11,11 +11,71 @@
 
 <body>
   <div class="root">
-    <header class="header"></header>
+    <header class="header" id="home">
+      <!-- Navigation -->
+      <nav class="nav" id="nav">
+        <div class="nav__content">
+          <!-- Logo And Hamburger Menu -->
+          <div class="nav__header">
+            <button class="nav__burger-menu"></button>
+            <a href="#home">
+              <img class="nav__logo" src="./images/logo.svg" alt="Логотип: Университет ИТМО">
```

Comment:
Could you try to reload this SVG to the project. Now it's compressed and looked very uncleared